packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -58,6 +51,10 @@ export default class TableResize implements EditorPlugin {
     private inserter: HTMLDivElement;
     private isRTL: boolean;
 
+    private isReadyForResizingWidth: boolean = false;
+    private isReadyForResizingHeight: boolean = false;
+    private isReadyForResizingTable: boolean = false;
+
```

Comment:
I don't like the idea to have these variables. It makes us always need to manage the flow that we need to reset it to false. If any place we missed to reset, there will be wrong result after that.

I prefer always call preprocessTable, and inside the function we can do some check to determine if we need to preprocess, such as if the table has width/height attribute, if each cell wdith/height are in "px", if the table as border box, ...

Reply:
fixed